versioned_docs/version-8.7/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -132,17 +132,92 @@ steps:
 
 Identity uses feature flag environment variables to enable and disable features; the supported flags are:
 
-| Environment variable         | Description                                                     | Default value |
-| ---------------------------- | --------------------------------------------------------------- | ------------- |
-| RESOURCE_PERMISSIONS_ENABLED | Controls the resource authorizations feature.                   | false         |
-| MULTITENANCY_ENABLED         | Controls the multi tenancy feature.                             | false         |
-| USER_RESTRICTIONS_ENABLED    | Controls the user task access restrictions feature in Tasklist. | true          |
+| Environment variable           | Description                                                     | Default value |
+| ------------------------------ | --------------------------------------------------------------- | ------------- |
+| `RESOURCE_PERMISSIONS_ENABLED` | Controls the resource authorizations feature.                   | `false`       |
+| `USER_RESTRICTIONS_ENABLED`    | Controls the user task access restrictions feature in Tasklist. | `true`        |
 
 :::note
 Setting either of the feature flags to `true` requires a database connection. To configure a database
 connection, see [database configuration](#database-configuration).
 :::
 
+## Multi-tenancy
+
+Multi-tenancy in the context of Camunda 8 refers to the ability of Camunda 8 to serve multiple distinct tenants or clients within a single installation. For more information, including additional configuration requirements, see the [multi-tenancy documentation](/docs/self-managed/operational-guides/configure-multi-tenancy.md).
+
+The following configuration is required to enable multi-tenancy in Identity:
+
+| Environment variable                    | Description                                                                                                               | Default value |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `CAMUNDA_IDENTITY_MULTITENANCY_ENABLED` | Controls the multi-tenancy feature within Identity. This can be set in helm via `camunda.tasklist.multi-tenancy.enabled`. | `false`       |
```

Comment:
> This can be set in helm via `camunda.tasklist.multi-tenancy.enabled`.

This seems to be wrong. Why would someone enable multi-tenancy via a Tasklist configuration property in Identity via helm? 😃 

In Helm, it would be `global.multitenancy.enabled`. But it sets it globally (identity, tasklist, operate, ...) for the entire C8 installed via helm. Given that this section focuses on Identity only, does it make sense to document the helm configuration at all here?